### PR TITLE
Update native data-control guide

### PR DESCRIPTION
**Pull request by kilig**
Update native data-control guide for access control with privileges

### CODE REVIEW

File: docs/application/native/guides/app-management/data-control.md [hunk 1]
```diff
@@ -46,11 +46,13 @@ To enable your application to use the data control functionality:
 
 2. For the provider, in the Tizen Studio, double-click **tizen-manifest.xml**, and in the manifest editor, go to **Advanced > Data Control**, and click **+** to add the provider details. Add the **Read** and **Write** access rights to both **SQL** and **Map** types.
 
-   The following code example shows how the `<datacontrol>` element is consequently added to the `tizen-manifest.xml` file:
+    You can set the data access to be trusted, allowing other applications signed with the same certificate to access the data. You can also define privileges to restrict access to applications having the specific privileges.
```
Comment (surbhig12) [inline docs/application/native/guides/app-management/data-control.md]: You can set the data access to be trusted, allowing other applications signed with the same certificate to access the data. You can also define privileges to restrict access to applications having the specific privileges.

[Surbhi] Modified the content to:

You can set the data access to trusted, to allow other applications that are signed with the same certificate to access the data.
Additionally, you can also define privileges to restrict access for applications having certain predefined privileges.

File: docs/application/native/guides/app-management/data-control.md [hunk 2]
```diff
@@ -46,11 +46,13 @@ To enable your application to use the data control functionality:
...
+
+    The following code example shows how the `<datacontrol>` elements are consequently added to the `tizen-manifest.xml` file:
```
Comment (surbhig12) [inline docs/application/native/guides/app-management/data-control.md]: The following code example shows how the `<datacontrol>` elements are consequently added to the `tizen-manifest.xml` file:

Please rephrase to:

The following sample code explains, how the `<datacontrol>` elements are consequently added to the `tizen-manifest.xml` file:

File: docs/application/native/guides/app-management/data-control.md [hunk 3]
```diff
@@ -1023,14 +1026,19 @@ The data model must be opened to the public to help other applications to use th
 
 - Data accessibility
 
-  Tizen native applications can control read and write access from other applications by defining data control accessibility.
+  - Tizen native applications can control read and write access from other applications by defining data control accessibility.
+- Trusted
+  - You can allow access from other applications signed with the same certificate by setting the trusted status for the data control.
+- Privileges
+  - Your provider application can restrict access to applications that have specific privileges defined.
```
Comment (surbhig12) [inline docs/application/native/guides/app-management/data-control.md]: Your provider application can restrict access to applications that have specific privileges defined.

Please rephrase to:

Your provider application can restrict access to applications having certain predefined privileges.

File: docs/application/native/guides/app-management/data-control.md [hunk 4]
```diff
@@ -46,7 +46,7 @@ To enable your application to use the data control functionality:
 
 2. For the provider, in the Tizen Studio, double-click **tizen-manifest.xml**, and in the manifest editor, go to **Advanced > Data Control**, and click **+** to add the provider details. Add the **Read** and **Write** access rights to both **SQL** and **Map** types.
 
-    You can set the data access to trusted, to allow other applications that are signed with the same certificate to access the data. Additionally, you can also define privileges to restrict access for applications having certain predefined privileges.
```
Comment (kilig) [inline docs/application/native/guides/app-management/data-control.md]: It is possible to use app-defined privileges also

File: docs/application/native/guides/app-management/data-control.md [hunk 5]
```diff
@@ -1030,7 +1030,7 @@ The data model must be opened to the public to help other applications to use th
 - Trusted
   - You can allow access from other applications signed with the same certificate by setting the trusted status for the data control.
 - Privileges
-  - Your provider application can restrict access to applications having certain predefined privileges.
```
Comment (kilig) [inline docs/application/native/guides/app-management/data-control.md]: It is possible to use app-defined privileges also

Comment (surbhig12) [reply docs/application/native/guides/app-management/data-control.md]: Thank you for the explanation.

I accept the comment.

File: docs/application/native/guides/app-management/data-control.md (hunk 4 above)
Comment (surbhig12) [reply docs/application/native/guides/app-management/data-control.md]: Thank you for the explanation.

I accept the comment.

**Comment by kilig** [comment]
I'm unfamiliar with Github. 
Should I do PR again?

**Comment by surbhig12** [comment]
Hello Kilig,

I have fixed the issue and committed the file (commit 51a0ea7f9e480524a128407530c189f2a0855f37).
You do not have to do a PR again.

Regards,
Surbhi

**Comment by kilig** [comment]
Thank you for your response

**Comment by surbhig12** [comment]
You are welcome :)

**Comment by mijin85cho** [comment]
Hello Kilig.
It looks like this PR can be published on Tizen Developer Site right now also.

It we should wait to apply this update on Tizen Developer Site, please let me know.
Otherwise we will update this update on Tizen Developer  Site, too.

**Comment by kilig** [comment]
It is already merged. Therefore, you should update this update on Tizen Developer Site.
Thanks

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**